pkg/gds/gds_test.go

```diff
@@ -105,6 +106,59 @@ func (s *gdsTestSuite) TestRegister() {
 	require.Error(err)
 }
 
+// TestVerification tests that the Verification RPC returns the correct status
+// information for a VASP.
+func (s *gdsTestSuite) TestVerification() {
+	s.LoadFullFixtures()
+	require := s.Require()
+	ctx := context.Background()
+
+	id := "d9da630e-41aa-11ec-9d29-acde48001122"
+
+	// The reference fixture doesn't contain the updated timestamp, so we retrieve the
+	// real VASP object here for comparison purposes.
+	vasp, err := s.svc.GetStore().RetrieveVASP(id)
+	require.NoError(err)
```

Comment:
The problem here is that when `generateDB` creates the VASPs in the store the `lastUpdated` timestamp gets updated, meaning the field on the VASP is different than the field in the reference fixture.